DMCompiler/DM/Visitors/DMProcBuilder.cs

```diff
@@ -116,13 +129,11 @@ class DMProcBuilder {
                 case DMASTProcStatementVarDeclaration varDeclaration: ProcessStatementVarDeclaration(varDeclaration); break;
                 case DMASTProcStatementTryCatch tryCatch: ProcessStatementTryCatch(tryCatch); break;
                 case DMASTProcStatementThrow dmThrow: ProcessStatementThrow(dmThrow); break;
-                case DMASTProcStatementMultipleVarDeclarations multipleVarDeclarations: {
-                    foreach (DMASTProcStatementVarDeclaration varDeclaration in multipleVarDeclarations.VarDeclarations) {
-                        ProcessStatementVarDeclaration(varDeclaration);
-                    }
-
+                //FIXME: Is there a more generic way of doing this, where Aggregate doesn't need every possible type state specified here?
+                case DMASTAggregate<DMASTProcStatementVarDeclaration> polyvar:
```

Comment:
You would still need a case for each type of aggregate, but you could have a generic `ProcessAggregate<T>()` method for processing them.
```cs
public void ProcessAggregate<T>(DMASTAggregate<T> aggregate) where T : DMASTProcStatement {
    foreach (var statement in aggregate.Statements)
        ProcessStatement(statement);
}
```

`polyvar` should also be renamed to `polyVar`